app.js

```diff
@@ -8,18 +8,26 @@ const app = express();
 
 const allScans = [];
 
-function logScanned(uuid, fixture) {
-  // TRICK: fixture tells if the the uuid was found in database or not
-  if (!/^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$/i.test(uuid)) {
-    return;
-  }
+function logScanned(uuid, matches) {
+  let allMatches = matches;
```

Comment:
redundant

Reply:
This in particular was to avoid linter errors.